pkg/api/docs/definitions/dashboard_permissions.go

```diff
@@ -20,6 +24,35 @@ import (
 //
 // Updates permissions for a dashboard.
 //
+// Please refer to [updated API](#/dashboard_permissions/postDashboardPermissionsWithUid) instead
+//
+// This operation will remove existing permissions if they’re not included in the request.
+//
+// Deprecated: true
+//
+// Responses:
+// 200: okResponse
+// 400: badRequestError
+// 401: unauthorisedError
+// 403: forbiddenError
+// 404: notFoundError
+// 500: internalServerError
+
+// swagger:route GET /dashboards/uid/{uid}/permissions dashboard_permissions getDashboardPermissionsWithUid
+//
+// Gets all existing permissions for the given dashboard.
+//
+// Responses:
+// 200: getDashboardPermissionsResponse
+// 401: unauthorisedError
+// 403: forbiddenError
+// 404: notFoundError
+// 500: internalServerError
+
+// swagger:route POST /dashboards/uid/{uid}/permissions dashboard_permissions postDashboardPermissionsWithUid
```

Comment:
this endpoint expects also a body parameter (same as the `POST /dashboards/id/{DashboardID}/permissions`)
therefore you need to add it also [here](https://github.com/grafana/grafana/blob/a0553de8dd41bc657b4706760a6a230d00c0969a/pkg/api/docs/definitions/dashboard_permissions.go#L33)

Reply:
@papagian I think I understood, I pushed a new commit